src/highdicom/sr/templates.py

```diff
@@ -1362,7 +1365,15 @@ def __init__(
                 if not isinstance(evaluation, CodeContentItem):
                     raise TypeError(
                         'Items of argument "qualitative_evaluations" must have '
-                        'type CodeContentItem.'
+                        'type CodeContentItem or QualitativeEvaluation.'
+                    )
+                rel_type = evaluation.relationship_type
+                expected_rel_type = RelationshipTypeValues.CONTAINS
+                if rel_type is None or \
+                   RelationshipTypeValues(rel_type) != expected_rel_type:
```

Comment:
This shouldn't be necessary, since `evaluation.relationship_type` should already return a value of type `RelationshipTypeValues`

```suggestion
                if rel_type is None or rel_type != expected_rel_type:
```

Reply:
I thought so too but in fact it returns a `str`... bit of a mistake in my opinion as I would definitely expect a `RelationshipTypeValues` in line with other properties around the library. We could fix in a non backwards compatible way?

Reply:
Oh..that's bad. I would consider this a bug. The property should return a `RelationshipTypeValues` instance, similar to the other properties. Is this something we should fix?

Reply:
I think so, I have fixed this (and the `value_type` property) in 259ae46. This is not backwards compatible, but I doubt much if any user code is using these properties directly. We should probably add something in the release notes though